### PR TITLE
Test/PE 572 multifile bundle tests

### PR DESCRIPTION
**Pull request by jdaev**
Adds tests for multi file bundles

Currently covered:

- Bundle packer

### CODE REVIEW

**Comment by fedellen** [comment]
You've got an unresolved conflict with a test file here

File: test/utils/bundle_packer_test.dart [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        final actualResult = await binPacker.packItems([testItem]);
+        final expectedResult = [
+          [testItem]
+        ];
+        expect(
+          DeepCollectionEquality().equals(actualResult, expectedResult),
+          true,
+        );
+      });
+      test('Creates a new bundle when next item does not fit', () async {
```
Comment (matibat) [inline test/utils/bundle_packer_test.dart:42]: Missing new line
```suggestion

      test('Creates a new bundle when next item does not fit', () async {
```

File: test/utils/bundle_packer_test.dart [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+      });
+
+      test('Throws an exception if item exceeds max packing size', () async {
+        final binPacker = NextFitBundlePacker(maxBundleSize: 5);
+        expect(
+          () async => await binPacker.packItems([TestSizedItem(size: 6)]),
+          throwsException,
+        );
+      });
+      test('Creates a single bundle of max size for one max size item',
```
Comment (matibat) [inline test/utils/bundle_packer_test.dart:29]: Missing new line
```suggestion

      test('Creates a single bundle of max size for one max size item',
```

**Review by matibat — CHANGES_REQUESTED** [review]
Left a couple of minor comments.
Requesting changes as the branch has conflicts with its base

**Review by matibat — APPROVED** [review]
LGTM

**Pull request merged**